packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -421,18 +421,18 @@ class RefreshIndicatorState extends State<RefreshIndicator> with TickerProviderS
     if (_mode == null) {
       assert(_dragOffset == null);
       assert(_isIndicatorAtTop == null);
-      return child;
+    } else {
+      assert(_dragOffset != null);
```

Comment:
This whole block should just be wrapped in an assert now\

Reply:
(including the line below that github won't let me select)

Reply:
Makes sense